target/linux/ramips/base-files/lib/ramips.sh

```diff
@@ -76,6 +76,9 @@ ramips_board_detect() {
 	*"BC2")
 		name="bc2"
 		;;
+	*"BL-R7628AE4")
```

Comment:
No need to add your board to this file, since you're using the devicetree based board name.